src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,615 @@
  * limitations under the License.
  */
 
-export const human38stats = {
-  assembly_stats: {
-    chromosome_count: 23,
-    genome_size: 3096649726,
-    total_sequence_length: 4537931177
+import { SpeciesStatsSection } from 'src/content/app/species/state/general/speciesGeneralHelper';
+
+type RawSpeciesStats = {
+  [genomeId: string]: {
+    [key in SpeciesStatsSection]: {
+      [key: string]: string;
+    };
+  };
+};
+
+export const sampleData: RawSpeciesStats = {
+  homo_sapiens_GCA_000001405_28: {
+    assembly_stats: {
+      scientific_name: 'Homo sapiens',
+      sex: 'NA',
+      breed_cultivar_strain: 'NA',
+      taxonomy_id: '9606',
+      assembly_name: 'GRCh38.p13',
+      assembly_accession: 'GCA_000001405.28',
+      assembly_date: '2019-02-28',
+      contig_n50: '56413054',
+      total_genome_length: '3272116950',
+      total_coding_sequence_length: '34317174',
+      total_gap_length: '161368351',
+      spanned_gaps: '661',
+      chromosomes: '25',
+      toplevel_sequences: '640',
+      component_sequences: '36734',
+      gc_percentage: '38.87'
+    },
+    coding_stats: {
+      scientific_name: 'Homo sapiens',
+      coding_genes: '20420',
+      average_genomic_span: '66885.16',
+      average_sequence_length: '3443.15',
+      average_CDS_length: '1139.02',
+      shortest_gene: '8',
+      longest_gene: '2473539',
```

Comment:
shortest_gene_**length**? longest_gene_**length**?

Reply:
That's done